src/components/accordion/accordion.tsx

```diff
@@ -145,7 +150,10 @@ export class EuiAccordion extends Component<
 
     return (
       <div className={classes} {...rest}>
-        <EuiFlexGroup gutterSize="none" alignItems="center">
+        <EuiFlexGroup
+          gutterSize="none"
+          alignItems="center"
+          responsive={responsiveHeader}>
```

Comment:
Thanks @sulemanof ! I actually think this just has been overlooked since there are few instances of accordions using the extra actions. However, I don't think there's a reason responsive should ever be true. The inner label will just wrap and everything else will just stay center aligned. How about we remove this new prop and just change this to be 
```suggestion
          responsive={false}>
```

Reply:
In fact, looking at @maryia-lapata's PR and the nested craziness of selectors in this file https://github.com/elastic/kibana/pull/40866/files#diff-1235e98f96e13b1964ee5a0e1b0ff94d makes me think we need to remove the EuiFlexGroup dependency all together from EuiAccordion.

Reply:
I think it's largely due to the negative margin issue with our flex system. It causes really dumb problems with anything that has an overflow applied (like accordion). https://github.com/elastic/eui/issues/937

We could either address it there or simply make something more specific in accordion like you mention. Likely though, people would just add flex groups back into it for more complicated layouts.